maintainers/maintainer-list.nix

```diff
@@ -1745,6 +1745,12 @@
     githubId = 28444296;
     name = "Benjamin Hougland";
   };
+  bigzilla = {
```

Comment:
`maintainers-list.nix` should be split on its own commit, with a message:

`maintainers: add bigzilla`